static/app/components/events/groupingInfo/groupingInfo.tsx

```diff
@@ -79,7 +77,7 @@ export default function GroupingInfo({
 
   return (
     <Fragment>
-      {hasStreamlinedUI && (
+      {(isSuccess || hasPerformanceGrouping) && (
```

Comment:
we still need the `hasStreamlinedUI` check here - this info is shown in a slightly different way in the old UI (there's a show/hide details button too). removing this means it'll show up twice

<img width="1099" alt="Screenshot 2025-03-24 at 9 40 46 AM" src="https://github.com/user-attachments/assets/69b3d16e-0df8-4d71-b5a8-4fedbcd852aa" />